docs/userguide/datetime/calendar/index.md

```diff
@@ -148,6 +148,7 @@ field by field.
 
 This is demonstrated in the following example:
 
+```C
```

Comment:
minor nit: C++ instead of C for the syntax highlighting.

Reply:
Done. Thx for the careful reading.